packages/unmock-core/src/backend/index.ts

```diff
@@ -101,7 +104,14 @@ export default class NodeBackend {
   }
 
   public get services(): ServiceStoreType {
-    return this.serviceStore;
+    return (this.serviceStore || NodeBackend.dummyStore).services;
+  }
+
+  public updateServices(newService: IObjectToService) {
```

Comment:
Couldn't this just be `updateService(newService: ServiceCore)` to keep the backend simple?

Reply:
It could -- but I'm not sure how it would keep the backend any simpler than it is right now. It's essentially replacing one type with another (`ServiceCore` is not used in the backend).

At any case, we could change the type but we'd have to make `serviceStore` either external or accessible from outside the backend, since a new `ServiceCore` is only created once we possibly retrieved an already existing one.

Reply:
Ok, looked just a bit weird to me have a instruction being passed through the interceptor. But I don't fully understand anymore what refers what so maybe it's fine.

Reply:
Added as a TODO note then :) will be part of refactoring